pkg/lib/verifier/rules/pcr_eventlog_equals_test.go

```diff
@@ -32,49 +30,21 @@ func TestPcrEventLogEqualsNoFault(t *testing.T) {
 		},
 	}
 
-	hostManifest.PcrManifest.PcrEventLogMapLinux.Sha256EventLogs = append(hostManifest.PcrManifest.PcrEventLogMapLinux.Sha256EventLogs, testHostManifestPcrEventLogEntry)
-
-	rule, err := NewPcrEventLogEquals(nil, &testHostManifestPcrEventLogEntry, uuid.New(), common.FlavorPartPlatform)
-
+	hostManifest.PcrManifest.PcrEventLogMap.Sha256EventLogs = append(hostManifest.PcrManifest.PcrEventLogMap.Sha256EventLogs, testHostManifestPcrEventLogEntry)
+	rule, err := NewPcrEventLogEquals(&testHostManifestPcrEventLogEntry, uuid.New(), common.FlavorPartPlatform)
 	result, err := rule.Apply(&hostManifest)
 	assert.NoError(t, err)
 	assert.NotNil(t, result)
 	assert.Equal(t, 0, len(result.Faults))
-	t.Logf("Equals rule verified for Intel Host Trust Policy")
-
-	//vmware
-	vmHostManifest := types.HostManifest{
```

Comment:
Was this unit test coverage dropped?

Reply:
Yes, Earlier there were 2 flows (Linux - dynamic and ESXI - hard coded) for flavor verification. Currently, we have changed ESXI flow to dynamic verification. So, separate unit test code for ESXI is no longer needed.